mgmtsystem/azure.py

```diff
@@ -355,6 +419,19 @@ def delete_stack(self, stack_name, resource_group=None):
             """.format(rg=resource_group or self.resource_group, stack=stack_name))
         return True
 
+    def delete_stack_by_date(self, days_old, resource_group=None):
+        self.logger.info("Removes a Deployment Stack resource created with Orchestration older "
+                         "than {} days".format(days_old))
+        self.run_script(
+            """
+            $Stack = Get-AzureRmResourceGroupDeployment -ResourceGroupName \"{rg}\"|
```

Comment:
I think we recently decided that everything between """  and """ should be indented one level.  The powershell character for line continuation is the tick mark. ` [tilde] key without the shift